domain/schema/testing/schema.go

```diff
@@ -29,3 +33,52 @@ func (s *SchemaApplier) Apply(c *gc.C, ctx context.Context, runner database.TxnR
 	c.Assert(err, gc.IsNil)
 	c.Check(changeSet.Post, gc.Equals, s.Schema.Len())
 }
+
+// DumpChangeLogState dumps the change log to the test log.
+func DumpChangeLogState(ctx context.Context, c *gc.C, runner database.TxnRunner) {
```

Comment:
This is not used at present, but it's OK to keep as a utility.

I would omit the context argument though, as it's only used in tests, and just supply `context.Background` to the query calls directly.

Reply:
Agreed. Will fix.